src/main/java/core/basesyntax/dao/impl/MessageDaoImpl.java

```diff
@@ -12,21 +14,65 @@ public MessageDaoImpl(SessionFactory sessionFactory) {
 
     @Override
     public Message create(Message entity) {
```

Comment:
Previously mentors asked to change "item", "entity" to "message", "user", etc. Don't know whether it needs to be done here...

Reply:
it will be nice, though it is not an absolute requirement